app/controllers/api_controller.rb

```diff
@@ -2,4 +2,22 @@
 
 class ApiController < ApplicationController
   # before_action -> { doorkeeper_authorize! :api }
+
+  def wrap_in_bundle(results)
+    # get just the FHIR resources, but then wrap it in an Entry.
+    resources = results.map { |r| wrap_in_entry(r.resource) }
+    FHIR::Bundle.new(type: 'searchset', entry: resources)
+  end
+
+  def wrap_in_entry(obj)
```

Comment:
The currated_model concern has a fhir_model method that does something similar to this.   Can that be leveraged?

Reply:
I don't think so, unless you see something I'm missing. If we're creating new `FHIR::Bundle`s then those take in Hashes, passing in FHIR::Models will cause it to crash. Plus this `wrap_in_entry` is being used by Profiles which don't include the CuratedModel concern.